Example/Example for Delirium/Modules/Dynamic Type/FontFamilyHeader.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FontFamilyHeader.swift
+//  Delirium
+//
+//  Created by Nikki Vergracht on 31/05/2017.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+class FontFamilyHeader: UITableViewHeaderFooterView {
+    
+    let label = UILabel()
+    
+    class var reuseIdentifier: String {
+        return String(describing: self)
+    }
+    
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.backgroundColor = .clear
+        label.textAlignment = .center
+        if #available(iOS 8.2, tvOS 8.2,  *) {
+            label.font = UIFont.systemFont(ofSize: 20, weight: UIFontWeightThin)
+        } else {
+            label.font = UIFont.systemFont(ofSize: 20)
+        }
+        contentView.addSubview(label)
+        contentView.addConstraint(NSLayoutConstraint(item: label, attribute: .leading, relatedBy: .equal, toItem: contentView, attribute: .leading, multiplier: 1.0, constant: 20))
```

Comment:
Actually, I just wrote normal constraints because there was still a reference to iOS 8. If that can be removed as a requirement, I can also update the constraints to `anchors`, which I prefer using for some time already 👍